src/matoya.h

```diff
@@ -1453,29 +1453,47 @@ MTY_WaitPtr(int32_t *sync);
 
 //- #module Audio
 //- #mbrief Simple audio playback and resampling.
-//- #mdetails This is a very minimal interface that assumes 2-channel, 16-bit signed PCM
+//- #mdetails This is a very minimal interface that supports multi-channel PCM audio
 //-   submitted by pushing to a queue. This module also includes a straightforward
-//-   resampler.
+//-   resampler for 2-channel, 16-bit signed PCM audio.
 
 typedef struct MTY_Audio MTY_Audio;
 typedef struct MTY_Resampler MTY_Resampler;
 
+/// @brief Audio sample formats. Currently only PCM formats.
+typedef enum {
+	MTY_AUDIO_SAMPLE_FORMAT_FLOAT   = 1, ///< 32-bit floating point.
+	MTY_AUDIO_SAMPLE_FORMAT_INT16   = 2, ///< 16-bit signed integer.
+} MTY_AudioSampleFormat;
```

Comment:
Why start this at 1?

Reply:
I think I did that to mimic `ParsecPCMFormat`:

![image](https://github.com/user-attachments/assets/7addff3b-8ebe-4c00-81ea-526146d5312d)

Reply:
Which presumably is done so that `0` can work as a sentinel value for "unknown format".